api_fhir_r4/models/coverage.py

```diff
@@ -35,8 +36,10 @@ class Coverage(DomainResource):
     relationship = Property('relationship', 'CodeableConcept')
     period = Property('period', 'Period')
     payor = Property('payor', 'Reference', required=True, count_max='*')  # referencing `Patient` | `RelatedPerson` | `Organization`
-    class_ = Property('class', 'CoverageClass', count_max='*')
+    classes = Property('classes', 'CoverageClass', count_max='*')
```

Comment:
@dragos-dobre, fyi we are not compliant here because the fHIR is section is called "class" but on my computer class_ didn't work (Property not found error)